main.js

```diff
@@ -0,0 +1,78 @@
+const app = () => {
+    const track = document.querySelector('.track');
+    const play = document.querySelector('.play');
+    const replay = document.querySelector(".replay");
+    const outline = document.querySelector('.moving-time-line circle');
+    const video = document.querySelector('.video-container video');
+    const timeSelect = document.querySelectorAll('.time input');
+
+    const sounds = document.querySelectorAll('.change-sound button');
+    const timeDisplay = document.querySelector('.time-display');
+    const outlineLength = outline.getTotalLength();  
+    
+    let fakeDuration = 600;
+    outline.style.strokeDasharray = outlineLength;
+    outline.style.strokeDashoffset = outlineLength;
+
+    sounds.forEach(sound => {
+        sound.addEventListener('click', function(){
+            track.src = this.getAttribute('data-sound');
+            video.src = this.getAttribute('data-video');
+            checkPlaying(track);
+        });
+    });
+
+    play.addEventListener("click", function() {
+        checkPlaying(track);
```

Comment:
if you write separate functions (handlers) for events, better to write all subscriptions (addEventListener) first and all handlers later